### ISSUE TITLE
[BUG] [pulasr] [Topic] non-persistent topic list get persistent topics result

### ISSUE THREAD

**Issue by archfish** [streamnative/pulsarctl#203]
```shell
./bin/pulsarctl topics list rental_car/promotions
```

```log
+--------------------------------------------+---------------+
|                 TOPIC NAME                 | PARTITIONED ? |
+--------------------------------------------+---------------+
| persistent://rental_car/promotions/changed | N             |
| persistent://rental_car/promotions/created | N             |
| persistent://rental_car/promotions/changed | N             |
| persistent://rental_car/promotions/created | N             |
+--------------------------------------------+---------------+
```

after try these rest apis

/admin/v2/persistent/:tenant/:namespace
/admin/v2/non-persistent/:tenant/:namespace

get same response cause the problem.

![image](https://user-images.githubusercontent.com/5317064/72881677-2cf7f080-3d3c-11ea-8a82-4fb2dba799bc.png)

**Comment by zymap** [comment]
@archfish  Which broker version you are using? It should be fixed in the 2.4.2.

**Comment by archfish** [comment]
> @archfish Which broker version you are using? It should be fixed in the 2.4.2.

I am using 2.4.1 . I will upgrade to newly version. Thanks!
